frontend/src/api/usePutData.js

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useState } from 'react';
+
+/**
+ * Custom hook updating data.
+ * @param {string} url - adress of request
+ * @param {state} requestBody - body od request made by user
+ * @returns { string object bool }
+ */
+const usePutData = async (url, requestBody) => {
+  const [data, setData] = useState(null);
+  const [error, setError] = useState(null);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    async function putData() {
+      try {
+        setIsLoading(true);
+        const response = await fetch(`${url}`, {
+          method: 'PUT',
+          headers: { 'Content-Type': 'application/json' },
```

Comment:
Let's extract it to the constants - file name can be eg. restResources.js
and the constant names can be REST_METHOD_PUT = 'PUT';
and the heders may be placed in DEFAULT_HEADERS = {'Content-Type': 'application/json'};